webapp/crashstats/supersearch/form_fields.py

```diff
@@ -206,7 +206,7 @@ class IsoDateTimeField(forms.DateTimeField):
     def to_python(self, value):
         if value:
             try:
-                return parse_isodate(value).replace(tzinfo=utc)
+                return parse_isodate(value).replace(tzinfo=datetime.timezone.utc)
```

Comment:
`django.utils.timezone.utc` is deprecated in Django 4.1 in favor of `datetime.timezone.utc` which it was an alias for since Django 4.0.

http://docs.djangoproject.com/en/5.0/releases/4.1/